README.md

```diff
@@ -785,50 +936,72 @@ mysql_database { 'mysql':
 ##### `user`
 
 The user for the database you're creating.
- 
+
 ##### `password`
 
 The password for $user for the database you're creating.
 
 ##### `dbname`
 
-The name of the database to create. Defaults to $name.
- 
+The name of the database to create.
+
+Defaults to $name.
```

Comment:
OK, I think $name is still a string, but it has a variable, so let's do double quotes around it.